### DISCUSSION TITLE
Azure Functionsで更新結果が反映されない

### DISCUSSION THREAD

**Discussion by ksera332** [1ft-seabass/ms-azure-hackathon-league-2021#51] [Q&A]
Azure ポータル上でFunctionのコードを更新し、念入りに保存をかけてから実行しても更新前のコードに戻される。

**Comment by papeMK2** [comment]
@ksera332 
ポータル上で検証してみました。

function.json を変更後、ポータル上で Function App を再読み込み(更新ボタンを押す、一度 Function の画面を閉じて再度その画面へ戻るなど)せずに、テストボタンを押すと function.json がロールバックする動きを確認しました。

function.json を書き換えた時点で実際にファイルの変更は行われており、function.json 書き換え後に、ポータル以外から API 呼び出しをした場合、
変更が反映されていました。

上記から、恐らくテストボタンのバグ or 仕様と思われます。

ファイル変更後は、一度更新ボタンを押してからテストすれば、恐らくこの問題は回避できるかと思いますので、お試しください。
